icons/chef.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Chef icon</title><path d="m 12.027228,-0.099609 c -5.0842374,0 -9.4577748,3.170815 -11.20718975,7.65369 L 2.7334608,8.319664 C 4.1548605,4.602157 7.817698,1.922725 12.027228,1.922725 c 2.241438,0 4.319082,0.766223 5.959158,1.968946 L 19.188681,2.251594 C 17.22059,0.775526 14.706019,-0.099609 12.027228,-0.099609 Z m 0,2.29568 c -4.1001915,0 -7.6534771,2.569667 -9.0748767,6.232504 L 4.8657738,9.139702 C 5.9591584,6.242234 8.7470747,4.219473 12.027228,4.219473 Z m 3.553499,0.711127 -0.765583,1.913423 c 2.405446,0.874707 4.26484,3.007233 4.756863,5.631356 h 2.076789 C 21.102104,7.007816 18.751541,4.109921 15.580727,2.907198 Z m -3.553499,1.585621 c -4.1001915,0 -7.4348002,3.334608 -7.4348002,7.4348 0,4.100191 3.3346087,7.434799 7.4348002,7.434799 3.608168,0 6.615189,-2.569666 7.325889,-5.959157 h -2.077858 c -0.7107,2.296107 -2.787917,3.936824 -5.248031,3.936824 -3.0068072,0 -5.4124665,-2.405659 -5.4124665,-5.412466 0,-3.006807 2.4056593,-5.412467 5.4124665,-5.412467 2.460114,0 4.537545,1.640718 5.193575,3.936825 h 2.077858 C 18.64263,7.007816 15.635396,4.492819 12.027228,4.492819 Z m 10.222716,1.093384 -1.748988,1.093385 c 0.7107,1.093384 1.202509,2.350989 1.421186,3.772389 H 24 C 23.781323,8.702562 23.179321,7.007603 22.249944,5.586203 Z m -10.277172,1.14784 c -2.7881298,0 -5.1391199,2.350776 -5.1391199,5.193576 0,0.765369 0.1644346,1.421826 0.4377807,2.077857 L 9.2393114,13.129915 C 9.0753038,12.74723 8.9659653,12.364972 8.9659653,11.927619 c 0,-1.640077 1.3667307,-3.006807 3.0068067,-3.006807 z m 1.913423,0.382258 -0.765583,2.023401 c 0.656031,0.273347 1.148481,0.710913 1.531165,1.312275 h 2.29568 C 16.510104,8.921239 15.362264,7.662993 13.886195,7.116301 Z M 0,11.927619 c 0,1.640076 0.32822886,3.280367 1.0389287,4.865774 L 2.8423722,15.973355 C 2.29568,14.661293 2.022334,13.294349 2.022334,11.927619 Z m 2.29568,0 c 0,4.100191 2.5154246,7.599021 6.1235932,9.020421 L 9.1848554,19.034617 C 6.3420561,17.941233 4.3190817,15.153103 4.3190817,11.927619 Z m 12.300574,1.475642 c -0.328015,0.601361 -0.819611,1.093597 -1.475642,1.312274 l 0.765583,2.023402 c 1.476069,-0.546692 2.623909,-1.804938 3.061262,-3.335676 z m 4.975753,0 c -0.7107,3.553499 -3.827273,6.232504 -7.544779,6.232504 v 2.023401 c 4.86556,0 8.910868,-3.554353 9.621568,-8.255905 z m 2.350135,0 c -0.765369,4.810891 -4.920016,8.529251 -9.894914,8.529251 -2.7334611,0 -5.2484589,-1.148266 -7.0525428,-2.952351 l -1.4211861,1.421186 c 2.1867687,2.186769 5.1933619,3.499044 8.5281839,3.499044 6.122952,0 11.207617,-4.592855 11.918317,-10.49713 z M 9.8949145,14.059932 8.3648172,15.590029 c 0.9293766,0.929377 2.2410108,1.531165 3.6624108,1.531165 v -2.186768 c -0.820039,0 -1.585621,-0.327802 -2.1323135,-0.874494 z" /></svg>
```

Comment:
Looks like the precision of the SVG is still at 6, could you lower that a bit (usually a precision of 3 is just right)?

Reply:
> Parece que a precisão do SVG ainda é de 6, você poderia diminuir isso um pouco (geralmente uma precisão de 3 é a correta)?

strange that I used the svgo and put the parameters

Reply:
> > Could you elaborate where you got the color from? I agree that this yellowish/orange color is a good one to represent the brand, but where exactly did you get the value `#F09820` from?
> > Also, as an aside, @sohammondal indicted in the issue that they're interested in adding the icon and I assigned them to the issue. Next time consider giving someone who shows interest in an issue some time to work on it and open a PR  That said all of your work is still very much appreciated!
> 
> I can close the PR without problem, for the contributor to make it :)

@ericcornelissen It would have been my first contribution to the open source community. :(

Reply:
> @ericcornelissen It would have been my first contribution to the open source community. :(

If you still want to work on this @sohammondal then go ahead and open a separate Pull Request. Since @JoaoPedroMoraes doesn't mind we can close this one. On the other hand most of the work has already been done, you can also choose another issue to work on (comment on the issue so it's clear, I'll assign you again). Usually one of the issues labeled [good first issue](https://github.com/simple-icons/simple-icons/issues?q=is%3Aopen+label%3A%22good+first+issue%22+label%3A%22new+icon%22) is a good one to start with :) (make sure there is no one working on it yet, this includes that there is no Pull Request related to it yet) Let me know what you want @sohammondal.

Reply:
@sohammondal Then do this PR, for I opened it because it had already begun before it was assigned to you. But new contributors are always welcome in the community. If you have any questions, please contact me via joaopedromoraes@outlook.com :smiley:

Reply:
@ericcornelissen  Hey, I have already completed and opened a separate [PR](https://github.com/simple-icons/simple-icons/pull/1510). If I want to work on a different issue, I will comment on that thread. I will leave it up to you to decide. Thanks. :)

@JoaoPedroMoraes Thanks for your effort. :)

Reply:
>  I opened it because it had already begun before it was assigned to you

That clarifies, no problem @JoaoPedroMoraes 😉 

> I have already completed and opened a separate PR. I will leave it up to you to decide which PR to approve.

For now I will close this PR and keep #1509 open.